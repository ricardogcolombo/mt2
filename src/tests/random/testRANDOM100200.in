100 200
1 5 48 55 51
1 100 46 69 48
1 48 45 35 4
1 68 60 2 32
1 57 108 1 74
1 55 109 81 121
1 22 16 59 101
1 74 22 59 0
1 52 71 40 128
1 68 110 19 44
1 59 31 2 105
1 96 15 22 126
1 86 54 72 48
1 73 62 33 117
1 80 24 35 89
1 22 44 31 106
1 13 34 70 109
1 24 26 18 95
1 9 47 49 29
1 28 41 68 53
1 95 29 79 69
1 58 125 35 14
1 66 115 84 86
1 18 66 17 42
1 82 45 9 40
1 39 78 53 72
1 4 25 43 73
1 68 28 61 89
1 100 21 98 12
1 94 113 100 5
1 5 1 78 119
1 61 107 21 99
1 91 86 35 71
1 50 17 44 95
1 44 105 21 28
1 2 130 94 53
1 32 115 6 128
1 15 75 9 84
1 67 17 84 92
1 12 6 54 14
1 80 54 38 56
1 68 64 27 8
1 82 87 93 77
1 99 42 1 83
1 20 39 27 117
1 85 107 35 105
1 35 20 33 21
1 27 129 96 80
1 92 113 27 126
1 47 5 48 21
1 39 41 9 61
1 37 15 43 59
1 49 9 23 87
1 59 110 30 22
1 67 94 24 49
1 6 27 34 8
1 24 76 82 93
1 10 11 58 84
1 30 31 51 66
1 71 99 33 24
1 28 115 27 82
1 92 3 89 99
1 36 115 14 77
1 66 85 23 5
1 40 66 71 70
1 37 111 32 105
1 50 114 74 24
1 57 68 49 8
1 93 37 54 109
1 17 112 71 130
1 35 124 4 29
1 5 4 83 130
1 5 60 88 128
1 44 20 72 17
1 93 31 94 102
1 94 55 3 15
1 36 8 74 60
1 59 110 18 45
1 71 40 51 82
1 26 81 23 31
1 15 56 53 71
1 93 3 81 24
1 17 52 10 125
1 99 35 55 94
1 56 30 42 128
1 67 38 9 48
1 64 77 65 73
1 36 104 65 100
1 51 16 66 44
1 58 76 83 98
1 28 38 46 58
1 59 126 46 94
1 14 22 100 126
1 71 61 55 101
1 95 0 83 93
1 95 10 43 51
1 39 72 22 126
1 77 18 64 51
1 78 79 84 7
1 22 14 34 78
1 68 113 66 4
1 41 14 37 3
1 52 20 62 58
1 51 98 96 27
1 45 11 33 125
1 40 16 79 42
1 84 45 63 119
1 82 73 85 29
1 65 85 100 96
1 9 111 70 59
1 21 123 97 35
1 2 19 96 98
1 26 110 11 86
1 76 106 6 8
1 70 33 4 9
1 93 14 10 7
1 9 122 42 40
1 53 1 66 38
1 26 63 43 117
1 67 89 82 92
1 18 99 40 84
1 75 70 11 8
1 30 14 62 55
1 57 72 35 31
1 79 92 66 77
1 11 110 50 1
1 11 25 24 116
1 24 108 42 32
1 88 118 52 40
1 44 17 93 1
1 69 16 50 78
1 82 124 49 40
1 57 124 76 119
1 55 70 35 46
1 69 41 97 27
1 74 36 14 82
1 42 72 73 111
1 20 26 54 65
1 10 33 88 82
1 61 99 39 102
1 44 96 12 24
1 64 59 85 52
1 80 57 73 127
1 59 125 61 7
1 98 5 10 112
1 22 51 11 96
1 7 13 65 14
1 95 10 31 39
1 86 0 8 127
1 23 65 4 85
1 22 99 63 119
1 79 21 6 90
1 86 76 93 75
1 4 90 65 99
1 65 127 18 72
1 4 24 74 47
1 6 37 9 47
1 28 108 5 69
1 31 120 17 11
1 37 4 99 12
1 61 15 10 105
1 7 54 37 57
1 85 62 6 4
1 58 121 84 55
1 20 70 76 93
1 97 40 38 12
1 73 4 71 6
1 27 10 9 95
1 100 18 14 34
1 98 74 27 1
1 83 103 62 73
1 88 14 2 42
1 91 60 64 89
1 24 4 90 62
1 64 120 100 55
1 29 121 40 72
1 47 121 25 101
1 60 104 14 29
1 55 26 8 2
1 24 93 96 113
1 13 32 20 104
1 61 84 10 52
1 45 31 41 45
1 67 103 22 95
1 60 87 61 127
1 13 115 49 124
1 87 72 73 43
1 6 77 56 12
1 23 27 25 30
1 97 102 72 41
1 94 61 65 8
1 100 3 97 121
1 8 71 68 7
1 17 16 100 94
1 66 5 25 72
1 96 0 46 104
1 21 92 83 14
1 94 119 62 17
1 49 62 53 94
1 19 4 3 85
