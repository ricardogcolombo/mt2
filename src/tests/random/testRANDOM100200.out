0.751829
0.726678
0.329214
0.705946
0.423996
0.495900
0.372923
0.181265
0.707117
0.235081
0.360427
0.759478
0.392442
0.342906
0.672839
0.602379
0.656665
0.363323
0.553257
0.569621
0.249662
0.679574
0.500416
0.469050
0.522301
0.574039
0.530360
0.528719
0.530304
0.570066
0.528641
0.546286
0.620769
0.366720
0.506477
0.364365
0.537700
0.442362
0.416744
0.659368
0.335687
0.526842
0.478104
0.547882
0.716223
0.179506
0.286481
0.428857
0.372876
0.428231
0.646884
0.767011
0.595494
0.418797
0.472651
0.503178
0.611896
0.609738
0.700656
0.733552
0.500000
0.346807
0.273280
0.408030
0.622240
0.678614
0.433038
0.252642
0.499669
0.143501
0.163176
0.553128
0.730451
0.425193
0.477112
0.421748
0.499253
0.524971
0.326144
0.507732
0.359902
0.371327
0.633679
0.417721
0.313844
0.525288
0.793070
0.524553
0.450698
0.537596
0.513773
0.809920
0.701772
0.493336
0.541877
0.575470
0.400591
0.739124
0.232308
0.569668
